docker/test/integration/features/kafka.feature

```diff
@@ -44,12 +44,9 @@ Feature: Sending data to using Kafka streaming platform using PublishKafka
       | PublishKafka   | Delivery Guarantee     | 1                                          |
       | PublishKafka   | Request Timeout        | 10 sec                                     |
       | PublishKafka   | Message Timeout        | 12 sec                                     |
-      | PublishKafka   | Security CA            | /tmp/resources/certs/ca-cert               |
```

Comment:
although we deprecated these properties in favor of the context service, shouldn't we keep testing the feature until we remove them?

Reply:
Makes sense, I added that version as well in 0cf6d007765d021d20573b0be61c6b793b60bf80